toxcore/DHT.c

```diff
@@ -1405,7 +1407,10 @@ static bool sent_getnode_to_node(DHT *dht, const uint8_t *public_key, IP_Port no
     }
 
     Node_format test;
-    memcpy(&test, data, sizeof(Node_format));
+
+    if (unpack_nodes(&test, 1, nullptr, data, sizeof(data), false) != 1) {
+        return false;
```

Comment:
If this ever fails, it's a hard bug (nothing outside the code influences this - it should never fail), so we should probably at least log an error here.

Reply:
I'll start a list of things we need to log once the logger is added to the DHT code